WeatherNotFound/src/main/java/me/learning/weathernotfound/presentation/WeatherNotFound.kt

```diff
@@ -78,7 +79,14 @@ class WeatherNotFound private constructor() {
 
         setInitCalledState(true)
 
-        OPEN_WEATHER_API_KEY = openWeatherApiKey
+        OPEN_WEATHER_API_KEY = context.packageManager
```

Comment:
Please create a function for this operation and return just an `String` as the result. I believe it will help the developer understand code more easily.